### PR TITLE
Task 246-248: DREAM

### PR DESCRIPTION
**Pull request by yeganehkordi**
Based on: https://arxiv.org/abs/1902.00164

### CODE REVIEW

File: tasks/README.md [hunk 1]
```diff
@@ -198,4 +198,7 @@ Name | Summary | Category
 `task205_remove_even_elements.json` | Given a list of integers remove all elements that are even | Answer Generation, Arithmetic
 `task206_collatz_conjecture.json` | Given a list of integers compute the next number in the *3n+1* problem | Answer Generation, Arithmetic
 `task207_max_element_lists.json` | Given a list of lists of integers compute the max value for each list | Answer Generation, Arithmetic
-`task208_combinations_of_list.json` | Given a list of integers of length *n* find all possible combinations,without replacement, of length *n-1* | Answer Generation, Combinatorics
\ No newline at end of file
+`task208_combinations_of_list.json` | Given a list of integers of length *n* find all possible combinations,without replacement, of length *n-1* | Answer Generation, Combinatorics
+`task246_dream_question_generation` | Writing a question based on a given conversation | Question Generation
```
Comment (Palipoor) [inline tasks/README.md]: Maybe change to:
`Given a conversation, generate a multiple-choice question based on it.`

**Comment by Palipoor** [comment]
Hi @yeganehkordi,
Everything seems ok to me, can you maybe explain that "w" and "m" stand for "woman" and "man" in task definitions? As I see they're referred to as "woman" and "man" in questions and answers, but there's no explanation.

File: tasks/README.md (hunk 1 above)
Comment (yeganehkordi) [reply tasks/README.md]: Sure!

**Comment by yeganehkordi** [comment]
Hi,
Thanks, it makes sense. I fixed it.

**Review by swarooprm — APPROVED** [review]
Looks good! Merging.
One comment: You can actually create more tasks using this dataset. A MCQ question can be split into answer generation and incorrect answer generation (look at task004 and task005)

**Pull request merged**